dev/deps/spark-deps-hadoop-2.6

```diff
@@ -93,7 +93,7 @@ jackson-core-asl-1.9.13.jar
 jackson-databind-2.6.7.1.jar
 jackson-jaxrs-1.9.13.jar
 jackson-mapper-asl-1.9.13.jar
-jackson-module-paranamer-2.6.7.jar
+jackson-module-paranamer-2.7.9.jar
```

Comment:
It's weird, yes, but this is what jackson-module-paranamer-2.6.7.1 pulls in:

https://repo1.maven.org/maven2/com/fasterxml/jackson/module/jackson-module-scala_2.11/2.6.7.1/jackson-module-scala_2.11-2.6.7.1.pom

Reply:
.. and I should say that it does look like Jackson support for Scala 2.12 really starts with 2.7.9, even though there is a special 2.6.7.1 release for Scala 2.12. It might become evident later we need to update Jackson, though it raises enough compatibility concerns that I wanted to hold off for now.